dracut-sshd.spec

```diff
@@ -0,0 +1,37 @@
+Name:       {{{ git_dir_name }}}
+Version:    {{{ git_dir_version }}}
+Release:    1%{?dist}
+Summary:    Add sshd to your initramfs
+
+License:    GPLv3+
+VCS:        {{{ git_dir_vcs }}}
+
+Source:     {{{ git_dir_pack }}}
+BuildArch:  noarch
+%define debug_package %{nil}
+Requires:   dracut-network
+
+%description
+Add sshd to your initramfs
+
+%prep
+{{{ git_dir_setup_macro }}}
+
+%build
+# nothing to do here
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}/usr/lib/dracut/modules.d/
+mv 46sshd %{buildroot}/usr/lib/dracut/modules.d/
```

Comment:
Can you change that to `cp -r ...`?

Such that your source tree isn't modified.